pkg/controller/clusterautoscaler/clusterautoscaler_controller.go

```diff
@@ -243,7 +276,8 @@ func (r *Reconciler) AutoscalerDeployment(ca *autoscalingv1alpha1.ClusterAutosca
 	}
 
 	annotations := map[string]string{
-		criticalPod: "",
+		criticalPod:                    "",
+		"release.openshift.io/version": r.config.ReleaseVersion,
```

Comment:
Ah, this is why.  The annotation should be on the Deployment, not the DeploymentSpec.metadata (I assumed it was setting on both).

That's good, we validated that the CVO waits!

Reply:
Updated.